src/Filters/molecules/TableLayoutToggle.tsx

```diff
@@ -0,0 +1,316 @@
+import Icon from '../../Icons/Icon';
+
+const Container = styled.div`
+  display: inline-block;
+  position: relative;
+`;
+
+const ContextActionBaseCSS = css`
+  -webkit-tap-highlight-color: transparent;
+  border: none;
+  background: none;
+  display: flex;
+  justify-content:left;
+  align-items:center;
+  flex-wrap: wrap;
+  cursor: pointer;
+  outline: none;
+  text-decoration: none;
+  padding: 0;
+`;
+
+const ContextIcon = styled.div`
+  ${({ theme }) => css`
+    ${theme.styles.global.mainMenu.iconBackground.default};
+    transition: background ${theme.animation.speed.fast} ${theme.animation.easing.primary.easeInOut};
+  `};
+  width: 30px;
+  height: 30px;
+  border-radius: 3px;
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  svg {
+    display: block;
+  }
+`;
+
+const ContextActionButton = styled.button<{ isActive?: boolean, isInnerContextButton?: boolean }>`
+  ${ContextActionBaseCSS}
+  ${ContextIcon}{
+    cursor: pointer;
+  }
+  &:hover {
+    div > svg > g {
+      stroke: hsl(0, 0%, 100%);
+    }
+  }
+  &:hover ${ContextIcon}{
+    ${({ theme }) => theme.styles.global.mainMenu.iconBackground.hover};
+    }
+  }
+  ${({ isActive }) => isActive && css`
+    ${ContextIcon} {
+      ${({ theme }) => theme.styles.global.mainMenu.iconBackground.hover};
+    }
+    &:hover ${ContextIcon}{
+      background-color: hsl(205, 100%, 72%);
+      cursor: pointer;
+    }
+  `}
+  ${({ isInnerContextButton }) => isInnerContextButton && css`
+    margin-right: 5px;
+  `}
+`;
+
+const ContentBox = styled.div<{ openState: IDropOpen, disabled: boolean, minWidth: number }>`
+  z-index: 100;
+  min-width: ${({ minWidth }) => minWidth}px;
+  position: absolute;
+  ${({ openState, disabled }) => openState && css`
+    display: ${openState.isOpen ? 'inline-block' : 'none'};
+    display: ${disabled && 'none'};
+    ${openState.position === 'bottom-right' && `
+      bottom: 0;
+      left: 0;
+      transform: translateY(calc(100% + 5px ));
+    `};
+    ${openState.position === 'bottom-left' && `
+      bottom: 0;
+      right: 0;
+      transform: translateY(calc(100% + 5px ));
+    `};
+    ${openState.position === 'top-left' && `
+      top: 0;
+      right: 0;
+      transform: translateY(calc( -100% - 5px ));
+    `};
+    ${openState.position === 'top-right' && `
+      top: 0;
+      left: 0;
+      transform: translateY(calc( -100% - 5px ));
+    `};
+  `};
+`;
+
+const ButtonWrapper = styled.div`
+  display: inline-block;
+`;
+
+const TopLine = styled.div`
+  ${({ theme }) => theme.styles.filters.dropdownContainer.topBorder};
+  height: 3px;
+  border-top-left-radius: 3px;
+  border-top-right-radius: 3px;
+`;
+
+const InnerBox = styled.div`
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  box-shadow: 0 4px 9px 0 hsla(204, 22%, 67%, 0.07);
+  border-right: solid 1px hsl(207, 16%, 86%);
+  border-bottom: solid 1px hsl(207, 16%, 86%);
+  border-left: solid 1px hsl(207, 16%, 86%);
```

Comment:
```suggestion
  border: solid 1px hsl(207,16%,86%);
  border-top: none;
```
We can refactor it like this.

Reply:
Done